docs/io.cozy.timeseries.md

```diff
@@ -42,3 +42,65 @@ These time series follow the [GeoJSON](https://geojson.org/) format. See [here](
 }
 
 ```
+
+## `io.cozy.timeseries.grades`
+This time series is used to represent a collection of grades in a given subject over a school period.
+
+It uses additional attributes :
+- `subject`: {string} The school subject these grades are referring to 
+- `title`: {string} The period (trimester or semester) name
+- `aggregation`: {Array} Grades averages over this period
+	- `avgGrades`: {number} Overall average of student grades over this period
+	- `avgClass`: {number} Overall average of class grades over this period
+	- `maxClass`: {number} Highest student overall average in the class
+	- `minClass`: {number} Lowest student overall average in the class
+- `series`: {Array} Contains the grades
+	- `id`: {string} Unique identifier for the grade
+	- `label`: {string} Grade label
+	- `date`: {date} Date of the grade
+	- `value`: {Array}
+		- `student`: {number} Student grade
+		- `outOf`: {number} Grade notation highest limit
+		- `coef`: {number} Coefficient of this grade
+		- `classAverage`: {number} Class average for this grade
+		- `classMax`: {number} Highest grading value in the class
+		- `classMin`: {number} Lowest grading value in the class
+- `status`: {Array}
+	- `isBonus`: {boolean} Is the grade counted only if it enhances the student’s average ?
+	- `isOptional`: {boolean} Is the grade an optional subject ?
+
+### Example
+```json
+
+{
+  "_id": "62e5d66d6e11d19992b7efce794263f0",
+  "subject": "hisgeo",
+  "title": "Semestre 2",
+  "startDate": "2017-04-22T01:00:00-05:00",  "endDate": "2017-07-01T01:00:00-05:00",
```

Comment:
small typo here, `endDate` should be on a new line

Reply:
Fixed typo in this commit -> https://github.com/cozy/cozy-doctypes/commit/863eefeadd73fa19e4dbc393d4030b3ec37a7ac4